lib/discourse_algolia/indexer.rb

```diff
@@ -33,6 +33,12 @@ def process!(ids: nil)
     @index.delete_objects(ids) if ids.present?
   end
 
+  def queue_ids
+    ids = Discourse.redis.lrange(self::QUEUE_NAME, 0, QUEUE_SIZE - 1)
+    Discourse.redis.ltrim(self::QUEUE_NAME, ids.size, -1)
+    ids.map(&:to_i).uniq
+  end
+
   def queue_ids
     Discourse.redis.lpop(self.class::QUEUE_NAME, 100)&.map(&:to_i)&.uniq || []
   end
```

Comment:
Shouldn't you delete that version of the method? 🤔

Reply:
Of course. :facepalm: